submissions/dashakim/popup_css_html/index.html

```diff
@@ -0,0 +1,173 @@
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="nav__link">
+                                    <img src="src/search.png" class="nav__icon icon" alt="search">
+                                    <div class="popup__text text">Search</div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="nav__link">
+                                    <img src="src/youtube.png" class="nav__icon icon" alt="youtube">
+                                    <div class="popup__text text">Youtube</div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="nav__link">
+                                    <img src="src/play.png" class="nav__icon icon" alt="play">
+                                    <div class="popup__text text">Play</div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="nav__link">
+                                    <img src="src/kmail.png" class="nav__icon icon" alt="kmail">
+                                    <div class="popup__text text">Mail</div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="nav__link">
+                                    <img src="src/drive.png" class="nav__icon icon" alt="drive">
+                                    <div class="popup__text text">Drive</div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="nav__link">
+                                    <img src="src/calendar.png" class="nav__icon icon" alt="calendar">
+                                    <div class="popup__text text">Calendar</div>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="nav__link">
+                                    <img src="src/business.png" class="nav__icon icon" alt="business">
+                                    <div class="popup__text text">My Business</div>
+                                </a>
+                            </li>
+                            <input type="checkbox" id="checkboxMore" class="popup-checkoxMore" />
+                            <label for="checkboxMore" class="popup-labelMore">More</label>
+                            <hr class="horisontal-row">
```

Comment:
For this case better use `border-bottom`. Do not clog your markup with unnecessary tags.